### DISCUSSION TITLE
Change syntax highliting color scheme?

### DISCUSSION THREAD

**Discussion by rohan-sircar** [chipzoller/hugo-clarity#178] [Q&A] [answered]
Is there an option to change the syntax highlting color scheme? 

I tried changing the highlight style value in markup.toml 
```
[highlight]
    style = "fruity"
```

but it doesn't do anything.

**Comment by onweru** [comment, marked as answer]
@rohan-sircar, no. The syntax highlighting color scheme is set in [via this file](https://github.com/chipzoller/hugo-clarity/blob/master/assets/sass/_syntax.sass)

**Reply by rohan-sircar** [reply]
I see, thanks!
